src/components/top-bar/TopBar.svelte

```diff
@@ -61,21 +61,26 @@
       </div>
       {#if $walletData}
         <span class="flex flex-grow"></span>
-        <span class="rounded-xl border-green border px-3 py-3 h-10 flex items-center justify-center text-center text-sm font-bold text-white">
-          {$walletData.account}
+        <span class="wallet-data">
+          {#if $walletData.name}
+            {$walletData.name}
+            {$walletData.avatar}
```

Comment:
`$walletData.avatar` is the url of the image. Will this render it properly?
Also could you check for `null`?
![2022-02-14 17 12 03 localhost 5de9815871ad](https://user-images.githubusercontent.com/93085583/153913147-e05bbb6a-9368-4bec-aa89-dff7985dc7a7.png)